octoprint_mrbeam/software_update_information.py

```diff
@@ -479,35 +481,36 @@ def _get_curent_version(input_moduleconfig, module_id, plugin):
     return current_version
 
 
-def _generate_config_of_beamos(moduleconfig, beamos_date, tierversion):
+def _generate_config_of_beamos(moduleconfig, beamos_version, tierversion):
     """
-    generates the config for the given beamos_date of the tierversion
+    generates the config for the given beamos_version of the tierversion
     Args:
         moduleconfig: update config of the module
-        beamos_date: date of the beamos
+        beamos_version: version of the beamos
         tierversion: software tier
 
     Returns:
         beamos config of the tierversion
     """
-    if "beamos_date" not in moduleconfig:
+    if "beamos_version" not in moduleconfig:
```

Comment:
Probably it would be good if we add a debug log  if we enter the if

Reply:
I can add one